main.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+class rs_m
+{
+public:
+    vector<double> r; 
+
+    void setResort(vector<double> r)
+    {
+        this->r = r;
+    }
+
+    //getting the size of the resort
+    double GetResortSize() 
+    {
+        int len = r.size();
+        return len;
+    }
+
+    int show()

```

Comment:
The function name follows the conventions, awesome! However, it's a bit unclear what is should do, since it's part of the `ResortManagement` class, could you add a more descriptive name?